CONTRIBUTING.rst

```diff
@@ -0,0 +1,175 @@
+=================
+How to Contribute
+=================
+
+First off, thanks for taking the time to contribute! ✨🎉
+
+Contributions are welcome, and they are greatly appreciated! Every
+little bit helps, and credit will always be given. The following is a
+set of guidelines for contributing to django-improved-user, hosted on
+`Github`_. These are mostly guidelines, not rules. Use your best
+judgment, and feel free to propose changes to this document in a pull
+request.
+
+Please remember that this is a volunteer-driven project. We will look at
+the issues and pull-requests as soon as possible.
+
+Code of Conduct
+---------------
+
+This project is subject to a `Code of Conduct`_. By participating, you
+are expected to uphold this code.
+
+Please be respectful to other developers.
+
+.. _Code of Conduct: https://github.com/jambonsw/django-improved-user/blob/development/CODE_OF_CONDUCT.md
+
+Types of Contributions
+----------------------
+
+You can contribute in many ways:
+
+Report Bugs
+~~~~~~~~~~~
+
+Please report bugs on the `Github issue tracker`_. Search the tracker to
+make sure someone else hasn't already reported the issue. If you find
+your the problem has already been reported, feel free to add more
+information if appropriate.  If you don't find the problem reported,
+please open a new issue, and follow the guidelines set forth in the text
+field.
+
+Fix Bugs
+~~~~~~~~
+
+Look through the `Github issue tracker`_ for bugs. Anything tagged with "bug"
+and "help wanted" is open to whoever wants to implement it. If someone
+has been assigned, or notes that it is claimed in in the comments,
+please reach out to them to work together on the issue to avoid
+duplication work. Note that, as volunteers, people sometime are unable
+to complete work they start, and that it is reasonable after a certain
+amount of time to assume they are no longer working on the issue. Use
+your best judgement to assess the situation.
+
+Write Documentation
+~~~~~~~~~~~~~~~~~~~
+
+The documentation aims to provide reference material, how-to guides, and
+a general tutorial for getting started with Django and
+django-improved-user. If you believe the documentation can be expanded
+or added to, your contribution would be welcome.
```

Comment:
Maybe use the past tense "welcomed" to match above verbs "expanded" and "added".